beacon-chain/rpc/beacon/blocks.go

```diff
@@ -178,7 +179,39 @@ func (bs *Server) GetChainHead(ctx context.Context, _ *ptypes.Empty) (*ethpb.Cha
 
 // StreamBlocks to clients every single time a block is received by the beacon node.
 func (bs *Server) StreamBlocks(_ *ptypes.Empty, stream ethpb.BeaconChain_StreamBlocksServer) error {
-	return status.Error(codes.Unimplemented, "Unimplemented")
+	blocksChannel := make(chan *feed.Event, 1)
+	blockSub := bs.BlockNotifier.BlockFeed().Subscribe(blocksChannel)
+	defer blockSub.Unsubscribe()
+	for {
+		select {
+		case event := <-blocksChannel:
+			if event.Type == blockfeed.ReceivedBlock {
+				data, ok := event.Data.(*blockfeed.ReceivedBlockData)
+				if !ok {
+					return status.Errorf(
+						codes.FailedPrecondition,
+						"Could not subscribe to block feed, received bad data: %v",
+						data,
+					)
+				}
+				if data.SignedBlock == nil {
+					return status.Error(
```

Comment:
maybe this should not error and stop the stream maybe 
```suggestion
					continue
```